src/index.js

```diff
@@ -236,7 +236,7 @@ const expressions = Object.assign(Object.create(null), {
             !!value &&
             typeof value === 'object' &&
             !!value.constructor &&
-            value.constructor.name === 'ObjectID' &&
+            value.constructor.name === 'ObjectId' &&
```

Comment:
Are you sure this won't cause an issue for us? If an ObjectId from a former version of the Mongo driver (that uses bson < 5) is passed, it will no longer match properly.

Reply:
@bradvogel from what I see [in Hound](https://hound.mixmax.com/?q=%5C%24type&i=nope&literal=nope&files=&excludeFiles=&repos=) we don't use `$type: objectId` at all in our code (not only via this package), so I believe it should not affect our code